src/Jackett.Common/Indexers/HDTorrents.cs

```diff
@@ -29,26 +40,23 @@ public class HDTorrents : BaseWebIndexer
             "https://hd-torrents.me/"
         };
 
-        private new ConfigurationDataBasicLogin configData
-        {
-            get => (ConfigurationDataBasicLogin)base.configData;
-            set => base.configData = value;
-        }
+        private new ConfigurationDataBasicLogin configData => (ConfigurationDataBasicLogin)base.configData;
 
         public HDTorrents(IIndexerConfigurationService configService, WebClient w, Logger l, IProtectionService ps)
-            : base(name: "HD-Torrents",
-                description: "HD-Torrents is a private torrent website with HD torrents and strict rules on their content.",
-                link: "https://hdts.ru/",// Of the accessible domains the .ru seems the most reliable.  https://hdts.ru | https://hd-torrents.org | https://hd-torrents.net | https://hd-torrents.me
-                caps: new TorznabCapabilities
-                {
-                    SupportsImdbMovieSearch = true,
-                    SupportsImdbTVSearch = true
-                },
-                configService: configService,
-                client: w,
-                logger: l,
-                p: ps,
-                configData: new ConfigurationDataBasicLogin())
+            : base("HD-Torrents",
+                   description: "HD-Torrents is a private torrent website with HD torrents and strict rules on their content.",
+                   link: "https://hdts.ru/", // Domain https://hdts.ru/ seems more reliable
+                   caps: new TorznabCapabilities
+                   {
+                       SupportsImdbMovieSearch = true,
+                       SupportsImdbTVSearch = true
```

Comment:
You recently commented out this line from all other trackers, we should do the same here I think.

Reply:
rebased